### ISSUE TITLE
Need `postinstall` script in `package.json` to build `dist/index.js`

### ISSUE THREAD

**Issue by darcyparker** [bcherny/json-schema-to-typescript#28]
Now that `dist/*` files aren't included in distribution, downstream repos that depend on `json-schema-to-typescript` are failing with 
```
>> Error: Cannot find module 'json-schema-to-typescript'
```
I think the cause is that https://github.com/bcherny/json-schema-to-typescript/blob/master/package.json#L5 refers to `dist/index.js` and this is not in the repo after `npm install json-schema-to-typescript`.

I speculate this can be resolved by adding `postinstall` script to your `package.json` like this: (See https://docs.npmjs.com/misc/scripts) (or something similar to this...)

```json
{
  "scripts": {
    "postinstall": "./node_modules/.bin/gulp dist"
  }
}
```

**Comment by darcyparker** [comment]
FYI: I have a WIP here: https://github.com/darcyparker/json-schema-to-typescript/tree/postInstallBuildDist

**Comment by bcherny** [comment]
Good catch - I should have tested this instead of assuming it worked as expected.

Fixed in 2.1.2 (ac26a7311e803217e50ccce410261436dd0d39f9).

**Comment by darcyparker** [comment]
Nice - Thanks.
